client/jetpack-connect/controller.js

```diff
@@ -381,3 +382,15 @@ export function redirectToLoginIfLoggedOut( context, next ) {
 
 	next();
 }
+
+export function redirectToSiteLessCheckoutIfLoggedOut( context, next ) {
+	const { type } = context.params;
```

Comment:
I could be wrong here, but I suspect that `type` might not be a real product slug. I believe that some Jetpack.com links use slugs that need to be mapped to real ones (e.g., `https://jetpack.com/upgrade/backup/` or `https://jetpack.com/upgrade/scan/`). If our checkout package knows how to deal with those types of slugs, disregard this comment.

Reply:
this did need fixed 😬